docs/js/getting-started.md

```diff
@@ -20,53 +20,296 @@ import LicenseBadge from '../../src/sap/sdk-js/LicenseBadge'
 
-To get started run
-```bash
+### Installation
+
+First run this command to install the CLI
+
+```shell
+npm install -g @sap-cloud-sdk/cli
+```
+
+### Usage
+
+```shell
+sap-cloud-sdk [COMMAND]
+```
+To get the CLI's version, run:
+```shell
+sap-cloud-sdk (-v|--version|version)
+```
+To get a list of all commands, run:
+```shell
+$ sap-cloud-sdk --help
+```
+
+### Create a new project
+
+To create a new project run the CLI's init command in the project folder.
+
+```shell
 $ sap-cloud-sdk init
 ```
-in the project folder.
 
 It will guide you through the initialization, create the necessary files and add necessary dependencies.
 If you run it in an empty folder, it will ask if you want to initialize a project using [@nest/cli](https://github.com/nestjs/nest-cli).
 
-To deploy to and run on Cloud Foundry, you need to
-1. Make sure that your app listens to port 8080
-2. Build your app if necessary
-3. Run [`sap-cloud-sdk package`](#sap-cloud-sdk-package)
-4. Push to Cloud Foundry (`cf push`)
+The CLI will already install all the necessary dependencies for the project, so this might take a minute. If everything worked correctly, you should see output like this:
+
+```shell
++---------------------------------------------------------------+
+| ✅ Init finished successfully.                                |
+|                                                               |
+| 🚀 Next steps:                                                |
+| - Run the application locally (`npm run start:dev`)           |
+| - Deploy your application (`npm run deploy`)                  |
+|                                                               |
+| 🔨 Consider setting up Jenkins to continuously build your app |
+| Use `sap-cloud-sdk add-cx-server` to create the setup script  |
++---------------------------------------------------------------+
+```
+
+## Project files and folders
+<!--not sure about this part-->
+
+The project contains the following files and folders, among others, to get you started with the SAP Cloud SDK for JavaScript:
+
+#### NPM / Project
+
+- **`package.json`**: Specifies dependencies, metadata and user-defined scripts. The application comes with some predefined scripts and dependencies, that will be explained in detail in the course of this group of tutorials.
+- **`.npmrc`**: The **`npm`** configuration file. The SAP Cloud SDK consists of some generic libraries, that are available as Open Source Software and service libraries for the whitelisted SAP S/4HANA APIs, referred to as the Virtual Data Model (VDM). In the scaffolding we specify the registry for the `@sap` scope, where the VDM libraries are published.
+
+#### TypeScript
+
+- **`tsconfig.json`**: Configuration file for `TypeScript`. This is not needed in the plain `JavaScript` version.
+- **`tslint.json`**: Configuration file for `tslint`, the de facto default linter for `TypeScript`.
```

Comment:
Unfortunately tslint is deprecated and we should update this as soon as possible ;)
```suggestion
- **`tslint.json`**: Configuration file for `tslint`.
```